css-dev/burf-tools/_tools-grunt.scss

```
@@ -0,0 +1,60 @@
+// =================================================================
+//
+// BURF Tools for Grunt builds (responsive-child-starter, etc.)
+//
+// A partial for those who only want the very basic of basics. Only
+// mixins and placeholders are supported in this partial, and only
+// pieces which are used will print. Useful for plugin development
+// where you want your dependencies and extra CSS to be at an
+// absolute minimum, or for development which will occur in another
+// pre-existing framework or environment with its own normalizing
+// partials and other opinionated stuff.
+//
+// =================================================================
+
+// =================================================================
+// Dependencies
+// =================================================================
+
+@import "math";
```

I think we also need to add the node modules import notation here as well, if we're avoiding using includePaths in our grunt config:

```suggestion
@import "~/mathsass/dist/math";
```

---

Same deal with grunt-sass over here - it won't recognize the `~` so we're stuck using the folder path name for now.